### ISSUE TITLE
Fire particles

### ISSUE THREAD

**Issue by sfederici** [nathan/phosphorus#558]
The project apparently finishes loading, but then it doesn't start at all.

http://scratch.mit.edu/projects/99521906/
http://phosphorus.github.io/#99521906
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

**Comment by htmlgames** [comment]
http://homepage.ntlworld.com/infinnerty/differences/UAT/itch/#99521906

works after a few seconds of black screen

**Comment by sfederici** [comment]
So what doesn't work is the initial speed test that should show a list viewer.

**Comment by sfederici** [comment]
No, wait. The original address http://scratch.mit.edu/projects/99521906 doesn't work at all. I can just see an empty white player forever

**Comment by htmlgames** [comment]
Note that hide / show list (viewer) isn't fully implemented yet

**Comment by sfederici** [comment]
Thanks. But do you have an updated version of phosphorus that solve some of its problems? If so, where can I get it? Thanks in advance.

**Comment by htmlgames** [comment]
I've added a link containing current progress..  [Zip]
Note that you will need a (local) webserver to play :)

**Comment by sfederici** [comment]
Thanks. Where can I find the link?

**Comment by htmlgames** [comment]
At the bottom of the project page is the [Zip] link:
http://homepage.ntlworld.com/infinnerty/differences/UAT/itch/
Direct download here: (though this may change in the future)
http://homepage.ntlworld.com/infinnerty/differences/UAT/itch.zip

**Comment by sfederici** [comment]
Thanks
